### PR TITLE
Book crud functionality for books service

### PR DESCRIPTION
**Pull request by AntonSkyrda**

### CODE REVIEW

File: books_service/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (aLEKS-e3) [inline books_service/admin.py:3]: register model in admin panel

File: books_service/models.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    COVER = [
+        ("HARD", "Hard cover"),
+        ("SOFT", "Soft cover")
+    ]
+
+    title = models.CharField(max_length=255)
+    author = models.CharField(max_length=255)
+    cover = models.CharField(max_length=10, choices=COVER)
+    inventory = models.PositiveIntegerField()
+    daily_fee = models.DecimalField(max_digits=6, decimal_places=2)
```
Comment (aLEKS-e3) [inline books_service/models.py:14]: add `__str__` method

File: library_management/urls.py [hunk 3]
```diff
@@ -1,22 +1,7 @@
...
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path("api/", include("books_service.urls", namespace="books_service"))
```
Comment (aLEKS-e3) [inline library_management/urls.py:6]: make `api/literature/` url since you've blocked an api path)

**Review by aLEKS-e3 — CHANGES_REQUESTED** [review]

**Review by kabust — CHANGES_REQUESTED** [review]
register your model in admin :)

**Review by kabust — APPROVED** [review]
Good job! 👍

File: books_service/models.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    def __str__(self):
+        return (f"{self.title} {self.author} {self.inventory}"
+                f" {self.inventory} {str(self.daily_fee)}")
```
Comment (aLEKS-e3) [inline books_service/models.py:18]: just leave title and author with daily fee)

**Review by aLEKS-e3 — CHANGES_REQUESTED** [review]

**Review by kabust — APPROVED** [review]
GJ!

**Review by aLEKS-e3 — APPROVED** [review]
nice!

**Pull request merged**